libs/libdrm/Makefile

```diff
@@ -45,9 +47,9 @@ endef
 
 MESON_ARGS += \
 	$(if $(CONFIG_LIBDRM_INTEL),-Dintel=true -Dlibkms=true,-Dintel=false -Dlibkms=false) \
-	-Dradeon=false \
+	$(if $(CONFIG_LIBDRM_NOUVEAU),-Dnouveau=true,-Dnouveau=false) \
```

Comment:
Could change to -Dnouveau=$(if $(CONFIG_LIBDRM_NOUVEAU),true,false) . Although maybe that's too pedantic...

Reply:
done